src/servant/tt_000/10E8.c

```diff
@@ -480,10 +482,16 @@ void func_80171ED4(s32 arg0) {
     dst = &g_Clut[0x1400];
     src = D_80170448;
 
+#ifdef VERSION_PC
+    // i exceeds the size of D_80170448
+    for (i = 0; i < 256 && i < 48; i++) {
+        *dst++ = *src++;
+    }
+#else
     for (i = 0; i < 256; i++) {
         *dst++ = *src++;
     }
-
+#endif
```

Comment:
Instead of reproducing the logic, what do you think about pulling out the upper bound into a macro that's set to 48 for PC, 256 otherwise.

(Odd that this is a inline copy instead of `memcpy`, `memmove`, etc.)